lib/media/track/es5/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = {
+  LocalAudioTrack: require('./localaudiotrack'),
+  LocalVideoTrack: require('./localvideotrack'),
+  LocalDataTrack: require('./localdatatrack'),
```

Comment:
```suggestion
  LocalDataTrack: require('./localdatatrack')
```

Reply:
@makarandp0 Nitpick: can you move LocalDataTrack above LocalVideoTrack to preserve alphabetical order?